library/std/src/sys/pal/unix/process/process_unix.rs

```diff
@@ -466,6 +454,7 @@ impl Command {
         envp: Option<&CStringArray>,
     ) -> io::Result<Option<Process>> {
         use crate::mem::MaybeUninit;
+        use crate::sys::weak::weak;
```

Comment:
What is this change for and how is it related?

Reply:
`./x build library --target=aarch64-apple-visions` throws a warning (that gets converted to an error) because the imported `weak` unused - see the commit that introduces it for details.